src/App.css

```diff
@@ -218,9 +214,11 @@ body {
   margin-bottom: 20px;
 }
 
-.add-book form {
+.custom-form {
   display: flex;
   justify-content: space-between;
+  gap: 3rem;
+  width: 100%;
 }
 
```

Comment:
- [ ] Good job so far on the design, but as mentioned in the project requirements your design should match the given design and you missed a lot of elements in your design.
- [ ] To find the design, [login to Zeplin](https://app.zeplin.io/login) using this credentials to login:
   -  username: `student@microverse.org`
   -  password: `jb077mL2dL0DfxJ`
   - Open the project *Bookstore CMS

Take a look at the picture below 👇🏾 👇🏾 

Your design

<img width="786" alt="Screenshot 2023-06-23 at 22 37 49" src="https://github.com/abiy006/Bookstore-React/assets/74501523/29b224dd-6d04-4fb7-9a8d-1a6b4ec1547e">

Given design

<img width="798" alt="Screenshot 2023-06-23 at 22 38 43" src="https://github.com/abiy006/Bookstore-React/assets/74501523/a40adb17-a587-4822-91a2-7c310025330c">

<img width="798" alt="Screenshot 2023-06-23 at 23 48 50" src="https://github.com/abiy006/Bookstore-React/assets/74501523/32bfcffb-15a7-49c5-af56-7407b3bb96b5">